config/settings.yml

```diff
@@ -0,0 +1,46 @@
+---
+:ems:
+  :ems_amazon:
+    :disabled_regions: []
+    # disable regions by their keys found in app/models/manageiq/providers/amazon/regions.rb e.g.
+    # uncomment the following to disable gov-west-1 region
```

Comment:
Note that this comment will not appear in Advanced Settings, so who is the audience for this comment?